test/e2e/tests/test_scenarios.py

```diff
@@ -128,3 +314,42 @@ class TestScenarios:
     """
     def test_scenario(self, scenario):
         runner.run(scenario)
+
+# assertTags returns AssertError if tags don't match
+def assertTags(expectedTags, latestTags):
+    for expectedTag in expectedTags:
+        tagKeyExist = False
+        for latestTag in latestTags:
+            if expectedTag == latestTag:
+                tagKeyExist = True
+                break
+        if tagKeyExist:
+            continue
+        assert expectedTag == None
+
+# assertParameters returns AssertError if parameters don't match
+def assertParameters(expectedParameters, latestParameters):
+    for expectedParameter in expectedParameters:
+        parameterExist = False
+        parameterName = expectedParameter.get("Name")
+        parameterValue = expectedParameter.get("Value")
+        for latestParameter in latestParameters:
+            if ("Name", parameterName) in latestParameter.items():
+                assert parameterValue == latestParameter.get("Value")
+                parameterExist = True
+                break
+        if parameterExist:
+            continue
+        assert expectedParameter == None
+
+# assertSubnets returns AssertError if subnets don't match
+def assertSubnets(expectedSubnets, latestSubnets):
+    for expectedSubnet in expectedSubnets:
+        subnetExist = False
+        for latestSubnet in latestSubnets:
+            if expectedSubnet.items() <= latestSubnet.items():
+                subnetExist = True
+                break
+        if subnetExist:
+            continue
+        assert expectedSubnet == None
```

Comment:
This isn't asserting the individual values?

Reply:
I think it does. In `expectedSubnet`, there are only one field `Identifier` (name). In latestSubnet from `describe_subnet_groups` called by `boto3`, there are other fields except `Identifier`. We only need to check if names match.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/memorydb/client/describe_subnet_groups.html